.github/workflows/language-reference.yaml

```diff
@@ -0,0 +1,61 @@
+name: language-reference-documentation
+
+on:
+  push:
+    branches:
+      - 'language-reference-stable'
+  pull_request:
+    branches:
+      - 'language-reference-stable'
+
+jobs:
+  build-and-push:
+    if: github.event_name == 'push'
```

Comment:
I never got right the way workflows are triggered by GH actions. Based on this YAML file, I believe this workflow is triggered on every push to the branch `language-reference-stable`, and also on every PR based off that branch? Why is this `if` condition necessary/useful?

Reply:
Ahh, this `if` is a mistake